.github/workflows/gradle-dependency-submission.yml

```diff
@@ -0,0 +1,18 @@
+# calculates dependencies for a Gradle build-target and submits the list to the Dependency Submission API.
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  update_gradle_dependencies:
+    runs-on: ubuntu-latest
+    permissions: # The Dependency Submission API requires write permission
+      contents: write
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+
+      - name: Gradle Dependency Submission
+        uses: mikepenz/gradle-dependency-submission@{latest}
```

Comment:
should this be fixed to a version?

Reply:
I don't really know. There is only one other workflow in this repo and I copied it from the reference repo if I remember correctly.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses doesn't mention `{latest}`. Recommends specifying the version, so I will add one in.